ALiAS FAQs added for blog and website (#150)
asetalias/asetalias.github.io: ALiAS_FAQs.md

FAQs
Q1. What is ASET AliAS?
Ans. ALiAS is a community, founded in 2010, which was developed to promote the use of Linux in the

> campus and get students familiar with the OS.

Reviewer: We don't do just OS anymore.

Reviewer: There are alot of activities, lot of programs, lot of groups for alot of things that ALiAS does. Try to find a better, more innovative answer. That begins like `It's not your regular Amity Club`

Q2. Why should I join ALiAS?

> Ans. ALiAS is by far, the best club inside Amity. Being an open source promoting community, we are

Reviewer: ```suggestion
Ans. ALiAS is by far, the largest open-source community in Amity University, we are
```

open to developers not only inside the campus, but also from various places. We have got assistance
from alumni and people from tech, near and far.
Q3. How is the community of AliAS?
And. We have got the most active developers in the campus. You will find people in nearly all domains,
Web Development, Android Development, Artificial Intelligence, backend, frontend, industry people,
etc. And best of all, we have got a great reach for people and further development.
Q4. How should I join AliAS?
And. Simple. Go and register yourself in http://asetalias.in/ . One of our developers will get you joined
to the group.
Q5. Having joined, where should I start?
And. We request you to drop a message, stating your name, course duration and your skills, if any. Also
let us know which field you want to work, and we will guide you thereafter. Have one plan at a time.

> Q6. Best OS?

Reviewer: Short answer, there is no best OS. 

I am not sure, why you chose Windows for designers. Considering Krita, Inkscape and Gimp all open-source software are literally matching Adobe in their specs and work. 

Why developers would go for Linux? Every question you ask? Ask another 'Why to your self'

Example. 

Statement: I like Pizza. 
Question: Why do I like Pizza
Now, write both of the answers. 

In the website version, we will give them straight answers. But, the blog will have a more detailed views of your own. I recommend you to make 2 PR's
